src/compiler/crystal/tools/doc/generator.cr

```diff
@@ -1,7 +1,13 @@
 class Crystal::Doc::Generator
+  class Options
+    property included_dirs = [] of String
+    property output_dir = "./docs"
```

Comment:
I think you can remove the `./`

Reply:
It was in there before, so I just copied it. But you're probably right, I don't see any reason as well.